crates/driver/src/boundary/liquidity/uniswap/v2.rs

```diff
@@ -43,7 +43,14 @@ pub fn to_domain(id: liquidity::Id, pool: ConstantProductOrder) -> liquidity::Li
         .downcast_ref::<uniswap_v2::Inner>()
         .expect("downcast uniswap settlement handler");
 
-    liquidity::Liquidity {
+    // Trading on Uniswap V2 pools where the reserves overflows `uint96`s does
+    // not work, so filter these pools out.
+    let max = (1_u128 << 96) - 1;
```

Comment:
Slight preference to using `pow` rather than `<<`.

Reply:
Changed.